src/Sentry/Laravel/Tracing/EventHandler.php

```diff
@@ -176,6 +184,12 @@ protected function queryExecutedHandler(DatabaseEvents\QueryExecuted $query): vo
             }
         }
 
+        if ($this->recordSqlBindings) {
+            $context->setData(array_merge($context->getData(), [
+                'db.sql.bindings' => $query->bindings
```

Comment:
@AbhiPrasad any better ideas for the naming here?

Reply:
Not solved in opentelemetry unfortunately - https://github.com/open-telemetry/opentelemetry-specification/pull/3092

The JS otel sdk has `db.postgresql.values` https://github.com/epsagon/opentelemetry-js-contrib/blob/1f6299d9d48ef2f7b206170827d3858e947474db/plugins/node/opentelemetry-instrumentation-pg/src/enums/AttributeNames.ts#L18

I would be open to `db.sql.values`, but that does make this more general, maybe the constrained definition of `db.sql.bindings` is good here.